test/multiverse/suites/active_record/Envfile

```diff
@@ -18,33 +13,50 @@ RAILS_VERSIONS = [
   ['3.2.0', 2.2, 2.2]
 ]
 
-def jbdc_version(activerecord_version)
+def jdbc_version(activerecord_version)
   return unless RUBY_PLATFORM.eql?('java') || activerecord_version.nil?
-  activerecord_version.slice('1')
+  # remove the '.' between the major and minor versions
+  activerecord_version_jdbc = activerecord_version.sub('.', '')
+  activerecord_version_jdbc
```

Comment:
My guess here is that this method is intended to determine which version of the `activerecord-jdbc-adapter` to use for a given ActiveRecord version.

From looking over the list of that gem's [versions](https://rubygems.org/gems/activerecord-jdbc-adapter/versions), it looks like for ActiveRecord version 5.2.0, there's an activerecord-jdbc-adapter version 52.0.

So following that logic, you'd want to convert the input of 5.2.0 into an output of 52.0 (not 520).

Unfortunately, this logic won't work for ActiveRecord versions 3.x and 4.x.

I'm wondering if we as a team should come up with a better way of deciding which gem version to use with each ActiveRecord version. Maybe it's as simple as "use this version for AR versions up to v5, this version for 5-6, and this version for 7+" or whatnot.

Reply:
Darn. That would be great to come up with something as a team. For this particular case, I think we would get for example, 52.0 vs 520, as `sub` just removes the first occurrence.